lib/config.js

```diff
@@ -47,7 +48,9 @@ module.exports = function (options) {
       log.warn('config', 'Authentication details missing from database URL')
```

Comment:
This has bitten me before. It would be great to provide more detailed help about how to proceed when this happens. If the user is trying to set up a couchdb against hoodie this message is not helpful.

Looks like the dockerfile contains enough information to solve this, but we shouldn't assume that the user knows that or that he wants to set up docker, IMO.